test/micro/org/openjdk/bench/javax/crypto/full/KeyPairGeneratorBench.java

```diff
@@ -50,6 +53,15 @@ public void setup() throws NoSuchAlgorithmException {
         }
     }
 
+    protected static Provider getInternalJce() {
+        try {
+            Class<?> dhClazz = Class.forName("com.sun.crypto.provider.DH");
+            return (Provider) (dhClazz.getField("PROVIDER").get(null));
```

Comment:
Nit: don't think you need the parens around `dhClazz.getField("PROVIDER").get(null)`